db/schema.rb

```diff
@@ -1037,6 +1060,15 @@
     t.index ["blocked_id", "blocker_id"], name: "index_user_blocks_on_blocked_id_and_blocker_id", unique: true
   end
 
+  create_table "user_collections", force: :cascade do |t|
+    t.datetime "created_at", null: false
+    t.string "title"
+    t.datetime "updated_at", null: false
+    t.bigint "user_id"
+    t.index ["user_id", "title"], name: "index_user_collections_on_user_id_and_title", unique: true
+    t.index ["user_id"], name: "index_user_collections_on_user_id"
+  end
+
```

Comment:
is this for testing/debugging purposes? not sure why we have these schema changes

Reply:
I removed these schema changes. Shouldn't be in here anymore